robottelo/constants/__init__.py

```diff
@@ -1656,6 +1656,31 @@ class Colored(Box):
     'Viewer',
 ]
 
+BOOKMARK_ENTITIES_SELECTION = [
+    {
+        'name': 'ActivationKey',
+        'controller': 'katello_activation_keys',
+        'session_name': 'activationkey',
+        'old_ui': True,
+    },
+    {'name': 'Errata', 'controller': 'katello_errata', 'session_name': 'errata', 'old_ui': True},
+    {'name': 'Host', 'controller': 'hosts', 'setup': entities.Host, 'session_name': 'host_new'},
+    {
+        'name': 'UserGroup',
+        'controller': 'usergroups',
+        'setup': entities.UserGroup,
+        'session_name': 'usergroup',
+    },
+    {
+        'name': 'Product',
+        'controller': 'katello_products',
+        'session_name': 'product',
+        'old_ui': True,
+    },
+    {'name': 'Repository', 'controller': 'katello_repositories', 'session_name': 'repository'},
+]
+
+# no longer used in favor of BOOKMARK_ENTITIES_SELECTION, kept for reference
 BOOKMARK_ENTITIES = [
```

Comment:
I would opt to just remove this. 1. No one will ever get the idea to use it again and 2. If we want to see what it was like before we have git :smile: